### DISCUSSION TITLE
lat,lon waypoint name

### DISCUSSION THREAD

**Discussion by OrxataEnFartons** [TUDelft-CNS-ATM/bluesky#414] [Q&A] [answered]
In the section of the wiki [Editting flight plans](https://github.com/TUDelft-CNS-ATM/bluesky/wiki/Editing-flight-plans) you say:

"_A lat-lon waypoint will be given a waypoint-name using the call sign of the aircraft followed by sequence number with leading zeros, **e.g. KL001**. This name can then be used as reference in the AT, AFTER ...._"

- Is that example ok? Because in the rest of examples in the same section you use KL001 as acid, so there I see nothing more than the acid. Maybe you meant something like KL001001, or KL00101? How many leading zeros then?
- When you say "will be given" you mean that that is done automatically, right?

Regards.

**Comment by jooste** [comment, marked as answer]
Hi @OrxataEnFartons,

I had to try this out to see :) If the callsign is `KL001`, the first lat/lon waypoint will be called `KL001001`. This is indeed done automatically, the next lat/lon waypoint will be `KL001002`, and so on.
